docs/setup.md

```diff
@@ -2,45 +2,50 @@
 
 This document outlines the setup process for the `Superposition Platform`.
 
-## Prerequisites
+## Installation
 
-* Docker installed and running.
-* Docker Compose installed.
-* [Follow installation instructions at Zero To Nix ↗](https://zero-to-nix.com/start/install) 
+- Install Rust using rustup
+    `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+- Install wasm-pack
+    `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
+- Install docker
+- make sure you have dependencies like libpq, openssl, libiconv,etc installed
```

Comment:
How does one do this in MacOS?  Any specific instructions needed for this ?

Reply:
@knutties I have written the steps to be platform independent, if necessary I can target ubuntu and macOS install instructions for libpq, openssl and libiconv